### PR TITLE
Add favicon, rating icons, and logo icon

### PR DESCRIPTION
**Pull request by tonycarpenter21**
What (if any) features are you implementing?
- Added favicon, logo icon, and ratings icons.

What (if anything) did you refactor?
- Nothing

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- No

Any other comments, questions, or concerns?
- No

### CODE REVIEW

File: src/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
           <div className='art-from-the'>
             Art From The 
           </div>
           <div className="logo-accent">
             Heart
           </div>
-        </Link>
-      </h1>
+        </h1>
+        <img src={heartIcon} alt="Art From The Heart" className="logo-icon" />
```
Comment (Jayquintana) [inline src/Navigation/Navigation.tsx:18]: good stuff tony!

**Review by Jayquintana — APPROVED** [review]
looks good !

File: src/ArtView/ArtView.tsx [hunk 2]
```diff
@@ -71,8 +73,12 @@ class ArtView extends Component<Props, State> {
...
     const loading = this.state.isLoading ? <Loading /> : <><img className="featured-art" src={`https://www.artic.edu/iiif/2/${this.state.randomImageId}/full/843,/0/default.jpg`} alt="painting of people at a park on a sunny day" />
     <div className="user-choices">
-      <button className="ratings-choice" onClick={() => this.findPiece(true)}>👍</button>  
-      <button className="ratings-choice" onClick={() => this.findPiece(false)}>👎</button>  
+      <button className="ratings-choice" onClick={() => this.findPiece(true)}>
+        <img src={thumbsUpIcon} alt="I like this art" />
+      </button>  
+      <button className="ratings-choice" onClick={() => this.findPiece(false)}>
+        <img src={thumbsDownIcon} alt="I don't like this art" />
+      </button>
```
Comment (mr-ryan12) [inline src/ArtView/ArtView.tsx:81]: Nice! Thank you for implementing this!

File: src/thumbs-down.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg width="73" height="74" viewBox="0 0 73 74" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M52.3687 4.00119H61.1644C63.0288 3.96822 64.8405 4.62096 66.2554 5.8355C67.6703 7.05005 68.5901 8.74185 68.84 10.5897V33.6497C68.5901 35.4975 67.6703 37.1893 66.2554 38.4039C64.8405 39.6184 63.0288 40.2712 61.1644 40.2382H52.3687M29.3087 46.8268V60.0038C29.3087 62.6249 30.35 65.1387 32.2034 66.9921C34.0567 68.8454 36.5705 69.8867 39.1916 69.8867L52.3687 40.2382V4.00119H15.2093C13.6203 3.98323 12.0785 4.54012 10.8677 5.56924C9.65697 6.59837 8.85896 8.0304 8.62071 9.60146L4.07461 39.2499C3.93128 40.1942 3.99497 41.1584 4.26126 42.0756C4.52755 42.9928 4.99008 43.8412 5.61679 44.5619C6.24351 45.2826 7.01943 45.8585 7.89079 46.2495C8.76215 46.6406 9.70812 46.8376 10.6632 46.8268H29.3087Z" stroke="white" stroke-width="6.58855" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```
Comment (mr-ryan12) [inline src/thumbs-down.svg:3]: Image looks great!

File: src/ArtView/ArtView.tsx [hunk 4]
```diff
@@ -71,8 +73,12 @@ class ArtView extends Component<Props, State> {
   render() {
     const loading = this.state.isLoading ? <Loading /> : <><img className="featured-art" src={`https://www.artic.edu/iiif/2/${this.state.randomImageId}/full/843,/0/default.jpg`} alt="painting of people at a park on a sunny day" />
     <div className="user-choices">
-      <button className="ratings-choice" onClick={() => this.findPiece(true)}>👍</button>  
-      <button className="ratings-choice" onClick={() => this.findPiece(false)}>👎</button>  
+      <button className="ratings-choice" onClick={() => this.findPiece(true)}>
```
Comment (StevieAmb) [inline src/ArtView/ArtView.tsx:76]: Awesome work!

**Review by StevieAmb — APPROVED** [review]

**Comment by mr-ryan12** [comment]
Thank you Tony! Next steps include creating our test suites!

**Pull request merged**